test/helpers/hasAccessTo.ts

```diff
@@ -0,0 +1,8 @@
+import { UserGroup } from "types/UserGroup"
+import { userAccess } from "utils/userPermissions"
+
+export const hasAccessToAll = userAccess({ groups: Object.values(UserGroup) })
+export const hasAccessToNone = userAccess({ groups: [] })
```

Comment:
I don't think we use this anywhere

Reply:
I've used these in the `listCorutCases` integration test now